includes/wp-cli/class-events.php

```diff
@@ -175,7 +179,7 @@ public function run_event( $args, $assoc_args ) {
 	 * @param array $assoc_args Array of flags.
 	 * @return array
 	 */
-	private function get_events( $args, $assoc_args ) {
+	private function retrieve_events( $args, $assoc_args ) {
```

Comment:
`get_events()` as a method named is used differently throughout the plugin, so I wanted to differentiate this some to make my code-searching life easier since it's private.